crates/iota/src/iota_commands.rs

```diff
@@ -27,6 +27,7 @@ use iota_genesis_builder::{SnapshotSource, SnapshotUrl};
 use iota_graphql_rpc::{
     config::ConnectionConfig, test_infra::cluster::start_graphql_server_with_fn_rpc,
 };
+use iota_grpc_api;
```

Comment:
Useless import?

Reply:
Because there is another `Config` you use `iota_grpc_api::Config`so either remove this here or import the config with an alias

Reply:
Thanks for pointing this out. Fixed at #8412